playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -132,6 +132,7 @@ EDXAPP_CAS_ATTRIBUTE_PACKAGE: ""
 EDXAPP_ENABLE_AUTO_AUTH: false
 # Settings for enabling and configuring third party authorization
 EDXAPP_ENABLE_THIRD_PARTY_AUTH: false
+EDXAPP_ENABLE_OAUTH2_PROVIDER: false
```

Comment:
OK, I see your point — sorry for being so dense :)

Currently, this setting exists, and is set to True, in these files
- lms/envs/dev.py
- lms/envs/devstack.py
- lms/envs/test.py

However, lms/envs/common.py sets it to False.

To enable ecommerce to authenticate against LMS, it needs to be True.

So for those of us needing ecommerce and working in, say, devstack, nothing's needed.  

However for the sandbox config., we need to set it somewhere; I chose /edx/etc/edxapp/cms.env.yaml, which is written by roles/edxapp/templates/cms.env.yaml.j2, with variables populated from roles/edxapp/defaults/main.yml.

If you prefer, I can split this off to a separate PR.